HelpSource/Classes/Array.schelp

```diff
@@ -404,3 +404,13 @@ code::
 [0.1, [\s_new, \default, -1, 1, 1, \freq, 1961]].asRawOSC;
 ::
 
+subsection:: BeLa
+
+method:: belaScope
+Send this Array's content to BeLa's Oscilloscope (see link::Classes/BelaScope:: for required setup)
+argument:: scopeChannel
+Bela's oscilloscope channel to start scoping on.
+
+argument:: server
+The server on which BelaScope is running. If not specified, it looks for the first server for which BelaScope was already initialized. If none is found, it attempts to initialize a link::Classes/BelaScope:: instance on link::Server#*default::.
+returns:: This Array.
```

Comment:
make sure all your files have an ending newline

Reply:
sure, sorry! thanks

Reply:
@brianlheim does the linting tool not process these files? Should it?

Reply:
the clang-format.py script only processes c++ files; clang-format doesn't know anything about schelp syntax.

there are a number of things that would be good to check in linting the entire codebase.